terraform/testcases/otlp_metric_amp_mock/otconfig.tpl

```diff
@@ -0,0 +1,30 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
```

Comment:
is there a default for this grpc_port value or does it have to be specified?

Reply:
4317